frontend/actions/actions.js

```diff
@@ -0,0 +1,20 @@
+const AppDispatcher = require("../dispatcher/dispatcher");
```

Comment:
I believe there's a way to set another lookup path in your webpack config so that you don't have to use relative paths in your require statements

In other words, instead of `require('../dispatcher/dispatcher')`, you could `require('dispatcher/dispatcher')`

Require also looks for `dispatcher/index.js` and `dispatcher/package.json` if it doesn't find `dispatcher.js` right away, so your require statement could just become `require('dispatcher')` if you rename `dispatcher.js` to `index.js`